deploy.sh

```diff
@@ -329,66 +75,3 @@ if [[ "$DEPLOY_NNS_DAPP" == "true" ]]; then
   OWN_CANISTER_URL="$(grep OWN_CANISTER_URL <"$CONFIG_FILE" | sed "s|VITE_OWN_CANISTER_URL=||g")"
```

Comment:
In principle this could return two or more lines.  You could fix that with `| head -n1` or like this:
```suggestion
  OWN_CANISTER_URL="$(sed -n 's|^VITE_OWN_CANISTER_URL=||;ta;b;:a;p;q')"
```
but this is a rather hypothetical situation and I'm fine with the code as is.

The sed:
```
-n == don't print by default
s == substitute; no g is needed as we will substitute only once.
ta == if there was a match, jump to a
b == branch == jump to <tag unspecified there for just skip the remaining commands for this line>
:a == the tag to jump to
p == print
q == quit
```